zello-sdk-sample/src/main/java/com/zello/sdk/sample/TalkActivity.java

```diff
@@ -26,6 +26,10 @@
 import android.widget.Toast;
 import android.widget.ToggleButton;
 
+import androidx.annotation.NonNull;
```

Comment:
Is the sdk project using the androidx `NonNull` annotation? Totally fine if so, just checking since we use the jetbrains `NotNull` in the main project

Reply:
If you only use the IDE specific NonNull annotation then I believe that the lint warning will not show when building your project in an automated CI/CD pipeline outside of the IDE. Because of this, the androidx NonNull annotation should probably be preferred.

Reply:
That's a good point – I should update the SDK project as well. I believe it doesn't currently use nullability annotations as it's been a while since we added anything new to it.